force_bdss/mco/optimizer_engines/space_sampling.py

```diff
@@ -40,12 +40,6 @@ class SpaceSampler(ABCHasStrictTraits):
     def __init__(self, dimension, resolution, **kwargs):
         super().__init__(dimension=dimension, resolution=resolution, **kwargs)
 
-    @abc.abstractmethod
```

Comment:
I didn't intend to do this. I did pull from master to begin, so I don't know whats happening with this.